spec/factories/events.rb

```diff
@@ -1,6 +1,6 @@
 FactoryBot.define do
   factory :event do
-    association(:move)
+    association(:entity)
```

Comment:
Should this be updated to `:eventable` ?

Reply:
yes it should be eventable 😊  - now fixed